### DISCUSSION TITLE
从localhost:7860/openpose_editor_index创建全新姿势，遇到问题，无法继续

### DISCUSSION THREAD

**Discussion by daixiaoluck** [huchenlei/sd-webui-openpose-editor#54] [Q&A] [answered]
大佬给了调整姿势的demo，youtube上至少有两个。谢谢大佬。
然而，大佬没给创建全新姿势的demo。我尝试在localhost:7860/openpose_editor_index做，点Download Image后，总是得到1591*1260的大图，比如下面这张：
![pose](https://github.com/huchenlei/sd-webui-openpose-editor/assets/10736725/11431c36-1b33-4146-9564-af335b4c75a5)
中心的黑色区域（似乎是canvas？），周围多了巨大灰色区域。
这种图放进controlnet，无法正常工作。我猜——纯猜测——周围灰色区域是罪魁祸首。
如果言语难以解释，麻烦大佬出个demo。我慢慢等。

**Comment by huchenlei** [comment, marked as answer]
See https://github.com/huchenlei/sd-webui-openpose-editor/issues/52. The issue has already been fixed. Please update your extension to lastest version.

**Reply by daixiaoluck** [reply]
身为菜鸟，不得不多问几句：
我之前用https://ghproxy.com/https://github.com/fkunn1326/openpose-editor.git ，我猜ghproxy没更新版本。
大佬意思是直接用https://github.com/fkunn1326/openpose-editor.git ？
我看不到v0.1.13这种版本，只能看到c9357715。我猜这是commit，但是master的commit记录，也没这hash。
![image](https://github.com/huchenlei/sd-webui-openpose-editor/assets/10736725/75b8d683-d1d1-4b71-beea-42d06682023d)

**Reply by huchenlei** [reply]
[fkunn1326/openpose-editor.git](https://ghproxy.com/https://github.com/fkunn1326/openpose-editor.git) is a completely different project. You need  to update `huchenlei/sd-webui-openpose-editor`.

**Reply by huchenlei** [reply]
Since ControlNet version v1.1.411, if you don't have local installation, we will use the remote up-to-date editor. You can remove/disable your local installation if you have stable connection to the remote editor here: https://huchenlei.github.io/sd-webui-openpose-editor/

如果你能稳定访问https://huchenlei.github.io/sd-webui-openpose-editor/ 的话，你可以把本地的插件禁用或者删除了。ControlNet插件会自动用online的版本。Online的版本是最新的。

**Reply by daixiaoluck** [reply]
成功了，谢谢。
另外，大佬讲过操作面部不可行。但是，（脖子和五官）整体改角度是个常见需求，似乎可行？比如脖子向左、右伸。个人建议，供大佬参考。
